src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const th = document.createElement('th');
+
+th.textContent = table.tHead.rows[0].cells[1].textContent;
+table.tHead.rows[0].cells[3].after(th);
+table.tFoot.rows[0].cells[3].after(th.cloneNode(true));
+
+for (const tr of table.tBodies[0].rows) {
+  const td = document.createElement('td');
+
+  td.textContent = tr.cells[1].textContent;
```

Comment:
Ensure that `tr.cells[1]` exists before accessing it. If a row has fewer than two cells, this will throw an error.